docs/source/using/profiles.adoc

```diff
@@ -8,29 +8,47 @@ include::variables.adoc[]
 
 toc::[]
 
-[[pofile-overview]]
+[[profile-overview]]
 == Overview
 
-{project} allows to create multiple instances of {project}.
-Each {project} instance can be created with different configurations and users can switch between different instances.
+The profile feature allows creating multiple instances of {project}. 
 
-[[creating-profile]]
+A profile is an instance of the {project} VM along with all of its configuration and state.
+
+Each {project} profile can be created with its own configuration (memory, CPU, disk size, add-ons, and so on) and completely independent of other profiles.
+Profiles share nothing between them.
+Things like cache, add-ons and persistent config are specific to a single profile.
+This means if you have set some specific configuration parameters for a profile, these settings do not apply to other profiles.
+
+The current profile being used is called the active profile.
+By default {project} commands will run against the active profile.
+You can run the xref:../command-ref/minishift_profile_list.adoc#[`minishift profile list`] command to findout the active profile.
+
+The profile feature introduced a global `--profile` flag and xref:../command-ref/minishift_profile.adoc#[commands] for listing, deleting and setting the active profile.
```

Comment:
I talk first about the commands and then introduce --global flag.

Reply:
I am wondering whether we need the --profile flag as part of the overview or whether this is also not a good candidate for a subsection "Executing commands against non active profile".

Reply:
> I talk first about the commands and then introduce --global flag.

IMO we should be talking about global flag first as it can be used with any other command. 

> I am wondering whether we need the --profile flag as part of the overview or whether this is also not a good candidate for a subsection "Executing commands against non active profile".

It would like to leave it in overview. Not sure if we need a dedicated section.